src/Filter/Persister/SessionFilterPersister.php

```diff
@@ -55,4 +59,22 @@ private function buildStorageKey(string $adminCode): string
     {
         return sprintf('%s.filter.parameters', $adminCode);
     }
+
+    /**
+     * TODO: Remove it when dropping support of Symfony < 5.3.
+     */
+    private function getSession(): SessionInterface
+    {
+        if (method_exists($this->requestStack, 'getSession')) {
+            return $this->requestStack->getSession();
```

Comment:
this method always returns a session? or can return null?

Reply:
See https://github.com/symfony/symfony/blob/5.3/src/Symfony/Component/HttpFoundation/RequestStack.php#L124

Reply:
Always a `SessionInterface`: https://github.com/symfony/symfony/blob/bf81015b7e094bc408d688f76b2f6f1e03f15b83/src/Symfony/Component/HttpFoundation/RequestStack.php#L119-L131, the `SessionNotFoundException` is also new.